Marlin/src/pins/stm32/pins_BTT_GTR_V1_0.h

```diff
@@ -0,0 +1,385 @@
+#if DISABLED(CUSTOM_SPI_PINS)
+  #define SDSS               PB12  
+#endif
+
+// HAL SPI1 pins group
+#if ENABLED(CUSTOM_SPI_PINS)
+    #define SDSS             PA4
+    #define SD_DETECT_PIN    PC4
+    #define LCD_SDSS         PA4
+
+    #define SCK_PIN          PA5
+    #define MISO_PIN         PA6
+    #define MOSI_PIN         PA7
+    #define SS_PIN           PA4   // Chip select for SD card used by Marlin
+ #endif 
+/**
+ *               _____                                             _____
+ *           NC | · · | GND                                    5V | · · | GND
+ *        RESET | · · | PB10(SD_DETECT)             (LCD_D7)  PG5 | · · | PG6  (LCD_D6)
+ *   (MOSI)PB15 | · · | PH10(BTN_EN2)               (LCD_D5)  PG7 | · · | PG8  (LCD_D4)
+ *  (SD_SS)PB12 | · · | PD10(BTN_EN1)               (LCD_RS)  PA8 | · · | PC10 (LCD_EN)
+ *    (SCK)PB13 | · · | PB14(MISO)                 (BTN_ENC) PA15 | · · | PC11  (BEEPER)
+ *               ￣￣                                               ￣￣
+ *               EXP2                                              EXP1
+ */
+
+//
+// LCDs and Controllers
+//
+#if HAS_SPI_LCD
+  #define BEEPER_PIN       PC11
+  #define BTN_ENC          PA15
+
+  #if ENABLED(CR10_STOCKDISPLAY)
+    #define LCD_PINS_RS    PA8
+
+    #define BTN_EN1        PD10
+    #define BTN_EN2        PH10
+
+    #define LCD_PINS_ENABLE PG7
+    #define LCD_PINS_D4    PG8
+
+    #undef ST7920_DELAY_1
+    #undef ST7920_DELAY_2
+    #undef ST7920_DELAY_3
+  #else
+
+    #define LCD_PINS_RS    PA8
+
+    #define BTN_EN1        PD10
+    #define BTN_EN2        PH10
+
+    #if DISABLED(CUSTOM_SPI_PINS)
+        #define SD_DETECT_PIN  PB10
+        #define LCD_SDSS   PB12
+    #endif
+    
+    #define LCD_PINS_ENABLE PC10
+    #define LCD_PINS_D4     PG8
+
+    #if ENABLED(FYSETC_MINI_12864)
+      #define DOGLCD_CS    PC10
+      #define DOGLCD_A0    PA8
+      //#define LCD_BACKLIGHT_PIN -1
+      #define LCD_RESET_PIN PG8   // Must be high or open for LCD to operate normally.
+      #if EITHER(FYSETC_MINI_12864_1_2, FYSETC_MINI_12864_2_0)
+        #ifndef RGB_LED_R_PIN
+          #define RGB_LED_R_PIN PG7
+        #endif
+        #ifndef RGB_LED_G_PIN
+          #define RGB_LED_G_PIN PG6
+        #endif
+        #ifndef RGB_LED_B_PIN
+          #define RGB_LED_B_PIN PG5
+        #endif
+      #elif ENABLED(FYSETC_MINI_12864_2_1)
+        #define NEOPIXEL_PIN    PF13
+      #endif
+    #endif // !FYSETC_MINI_12864
+
+    #if ENABLED(ULTIPANEL)
+      #define LCD_PINS_D5   PG7
+      #define LCD_PINS_D6   PG6
+      #define LCD_PINS_D7   PG5
+    #endif
+
+  #endif
+
+  // Alter timing for graphical display
+  #if HAS_GRAPHICAL_LCD
+    #define BOARD_ST7920_DELAY_1 DELAY_NS(96)
+    #define BOARD_ST7920_DELAY_2 DELAY_NS(48)
+    #define BOARD_ST7920_DELAY_3 DELAY_NS(600)
+  #endif
+
+
+//#define DOGLCD_CS PB12
+//#define DOGLCD_A0 PA8
+//#define LCD_PINS_DC PB14
+//#define DOGLCD_MOSI PB15
+
+#endif // HAS_SPI_LCD
```

Comment:
GTR pin files

Reply:
Thanks! There is no need to do a second Pull Request. Commits you add to your branch will be added to this one.